### PR TITLE
testing: 1.13.0 with new magma 2.6.2

### PR DESCRIPTION
**Pull request by ngam**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -73,7 +71,7 @@ outputs:
         # GPU requirements
         - cudnn                           # [cuda_compiler_version != "None"]
         - nccl                            # [cuda_compiler_version != "None"]
-        - magma                           # [cuda_compiler_version != "None"]
+        - magma 2.6.2                     # [cuda_compiler_version != "None"]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:74]: why do you have to be so specific.

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
lets wait for magma to be released, then rebuild.

**Comment by hmaarrfk** [comment]
We aren't as bad as Jax.... I just pushed to main to trigger a rebuild.

**Comment by hmaarrfk** [comment]
Sorry, as bad as arrow.

**Comment by h-vetinari** [comment]
> Sorry, as bad as arrow.

What are you saying here? 🤔

**Comment by hmaarrfk** [comment]
then number of builds to rebuild. ^_^ juts a joke.

**Comment by hmaarrfk** [comment]
arrow having 5 different branches, making it able to bog down azure ^_^

**Comment by h-vetinari** [comment]
> then number of builds to rebuild. ^_^ just a joke.

All good - sounds like you're gonna like https://github.com/conda-forge/arrow-cpp-feedstock/pull/875; we're now down to 16 jobs (20 once CUDA on aarch/ppc is reinstated), and that's _after_  doubling the build matrix due to openssl 3. _And_ it runs faster (no more restarts 🥳), despite running the full test suite now.





> arrow having 5 different branches, making it able to bog down azure ^_^

Yeah, I've set `max_parallel` on all branches, so it's not quite such a punch in the gut. 🙃

**Comment by hmaarrfk** [comment]
Great work! These science packages are really hard to compile!

**Pull request closed without merging**